index.md

```diff
@@ -9,55 +9,192 @@ iOS and Android from a single codebase.
+language and a declarative approach
+*   Prototype and iterate quickly
+    *   Experiment by changing code and reloading as your app runs
+    *   Fix crashes and continue debugging from where the app left off
+*   Create beautiful, highly-customized user experiences
+    *   Leverage a complete set of self-referential Material Design widgets
+    *   Realize whatever visual design you have in mind, without the limitations 
+of OEM widget sets
+
+## Core Principles
+
+Flutter includes a functional-reactive framework, a 2D rendering engine, ready-
+made widgets, and development tools. The components work together to help you 
+design, build, test, and debug apps. Everything is organized around a few core 
+principles. 
+
+### Everything's a Widget
+
+Widgets are the basic building blocks of every Flutter app. Each widget is an 
+immutable declaration of part of a user interface.  Unlike other frameworks that 
+separate views, controllers, layouts, and other properties, Flutter has a 
+consistent, unified object model: the widget.
+
+A widget can define:
+
+*   a structural element (like a button or menu)
+*   a stylistic element (like a font or color scheme)
+*   an aspect of layout (like padding)
+*   some business logic 
+*   and so on...
+
+Widgets form a hierarchy based on composition.  Each widget nests inside, and 
+inherits properties from, its parent.  There is no separate "application" object.
+Instead, the root widget serves this role.
+
+
+You can respond to events, like user interaction, by telling the framework to 
+replace a widget in the hierarchy with another widget.  The framework then 
+compares the new and old widgets and efficiently updates the user interface.
+
+#### Composition > inheritance
+
+Widgets are themselves often composed of many small, single-purpose widgets that
+combine to produce powerful effects.  For example, [Container](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/container.dart),
+a commonly-used widget, is made up of several widgets responsible for layout,
+painting, positioning, and sizing. Specifically, Container is made up of
+[LimitedBox](https://docs.flutter.io/flutter/widgets/LimitedBox-class.html),
+[ConstrainedBox](https://docs.flutter.io/flutter/widgets/ConstrainedBox-class.html),
+[Align](https://docs.flutter.io/flutter/widgets/Align-class.html),
+[Padding](https://docs.flutter.io/flutter/widgets/Padding-class.html),
+[DecoratedBox](https://docs.flutter.io/flutter/widgets/DecoratedBox-class.html),
+and [Transform](https://docs.flutter.io/flutter/widgets/Transform-class.html)
+widgets.  Rather than subclassing Container to produce a customized effect, you
+can compose these, and other, simple widgets in novel ways.
+
+The class hierarchy is shallow and broad to maximize the possible number of 
+combinations.
+
+![Widget class hierarchy](/images/whatisflutter/Widget_class_hierarchy.png){:style="max-width: 900px"}
```

Comment:
Looks like the fonts in KolourPaint (open source Linux drawing tool from kde.org) are ones that are included with the tool. There is nothing in the UI for adding fonts, no folder location, no documentation on adding fonts, no Google search results. Installing Roboto using your steps has no effect on what can be selected in the UI. I think this tool is limited in terms of fonts.

There is no "source" format - these are PNG images that can be edited.